test/build_optimized_size_test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
How do we trigger this? Add it to CI? Also is it possible to re-use https://github.com/pytorch/executorch/blob/main/test/build_size_test.sh?

Reply:
> How do we trigger this? Add it to CI?

As the description says, it is meant for development use.

> is it possible to re-use

No, that script wants to disable exceptions, which is appropriate in the embedded systems context but not the "everything else" context. Optimized kernels are not for the embedded systems context AIUI.